pages/linux/qm-guest-exec.md

```diff
@@ -0,0 +1,20 @@
+# qm guest exec
+
+> Execute a specific command via a guest agent.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Execute a specific command via a guest agent:
+
+`qm guest exec {{vm_id}} {{arg1 arg2 ...}}`
+
+- Execute a specific command via a guest agent asynchronously:
+
+`qm guest exec {{vm_id}} {{arg1 arg2 ...}} --synchronous 0`
+
+- Execute a specific command via a guest agent with a specified timeout of 10 seconds:
+
+`qm guest exec {{vm_id}} {{arg1 arg2 ...}} --timeout 10`
```

Comment:
I believe the timeout parameter can be changed, right?

```suggestion
`qm guest exec {{vm_id}} {{arg1 arg2 ...}} --timeout {{10}}`
```

Reply:
Yep, but @navarroaxel told me not to use braces. This is undefined by our style guide now. So there are no restrictions here.

Reply:
> Yep, but @navarroaxel told me not to use braces. This is undefined by our style guide now. So there are no restrictions here. 

I think @navarroaxel meant not using arguments for definite parameters like true, 1, etc which is necessary for the command to work. Not in this case, because the value can be changed.

Reply:
But true also can be changed to false. :) Formal definition should be provided for certain values.

Reply:
So just for a second, let me catch all the discussions you left before.

Reply:
This is an exception in most cases we specify `1` or `true` inside command arguments when `false` or `0` is possible but for some commands, if `false` or `0` is given it doesn't execute so the command argument `{{}}` is avoided. Also, Timeout should be a command argument as it can be any value and the same is used in this [page](https://github.com/tldr-pages/tldr/blob/main/pages/linux/qm-wait.md). Some references, [1](https://github.com/tldr-pages/tldr/blob/main/pages/linux/qm-snapshot.md), you can see more examples by going through other qm pages.

This must be documented in style guide.